### PR TITLE
handles PATCH /api/reviews/:review_id

### PR DESCRIPTION
**Pull request by SentinelProxima**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -90,4 +90,44 @@ describe('GET /api/users', () => {
...
+    });
+    it('returns 400 if non-numeric ID passed', () => {
+        return request(app).patch('/api/reviews/kitten').send({inc_votes: 1})
+        .expect(400)
+        .then((res) => {
+            const msg = JSON.parse(res.text);
+            expect(msg.msg).toBe("Invalid review ID");
+        });
+    });
 });
```
Comment (popatre) [inline __tests__/app.test.js:149]: I'd add two more tests

- [ ] 400 inc_vote not an integer on the patch object
- [ ] 200/400 inc_vote key missing - either respond with an error or make no change.

File: __tests__/app.test.js [hunk 2]
```diff
@@ -90,4 +90,44 @@ describe('GET /api/users', () => {
...
+        .expect(200)
+        .then((res) => {
+            expect(res.body.review.votes).toBe(3);
+        });
+    });
+    it('returns 404 if invalid (numeric) ID passed', () => {
+        return request(app).patch('/api/reviews/0').send({inc_votes: 1})
+        .expect(404)
+        .then((res) => {
+            const msg = JSON.parse(res.text);
```
Comment (popatre) [inline __tests__/app.test.js:121]: You dont need to parse the object to look as it, so could probably skip this out if you wanted.

File: controllers/reviews.controllers.js [hunk 3]
```diff
@@ -7,4 +7,15 @@ exports.getSingleReview = (req, res, next) => {
         res.status(200).send({ review });
     })
     .catch(next);
+};
+
+exports.patchReview = (req, res, next) => {
+    const review_id = req.params;
+    selectReview(review_id)
+    .then((review) => {
+        console.log(req.body);
```
Comment (popatre) [inline controllers/reviews.controllers.js]: This console log just needs removing once youre done

File: controllers/reviews.controllers.js [hunk 4]
```diff
@@ -7,4 +7,15 @@ exports.getSingleReview = (req, res, next) => {
...
+        review.votes += req.body.inc_votes;
```
Comment (popatre) [inline controllers/reviews.controllers.js]: You need to update the votes in the database. At the moment this just changes a value of the object, but the votes remain the same in the db. 

look at creating a model and how to update values in sql.

**Review by popatre — CHANGES_REQUESTED** [review]

**Review by Poonam-raj — APPROVED** [review]
Lovely stuff 🎉

**Pull request merged**